documentation/modules/ROOT/pages/021-initial-dir-and-files.adoc

```diff
@@ -133,7 +154,9 @@ EOF
 [.console-input]
 [source,console]
 ----
-export ADMIN_DEV_PASSWORD=<CHANGE_ME>
+export HISTCONTROL=ignoreboth
+ export ADMIN_DEV_PASSWORD=<CHANGE_ME>
+ export ADMIN_PRO_PASSWORD=<CHANGE_ME>
```

Comment:
Remove the first space character at each line

Reply:
space was added on purpose to avoid adding the command to the bash history